com.unity.render-pipelines.high-definition/Editor/BuildProcessors/HDRPPreprocessShaders.cs

```diff
@@ -560,15 +551,7 @@ public void OnProcessShader(Shader shader, ShaderSnippetData snippet, IList<Shad
                     for (int i = inputData.Count - 1; i >= inputShaderVariantCount; --i)
                         inputData.RemoveAt(i);
 
-                if (HDRenderPipelineGlobalSettings.instance.shaderVariantLogLevel != ShaderVariantLogLevel.Disabled)
-                {
-                    foreach (var hdAsset in hdPipelineAssets)
-                    {
-                        m_TotalVariantsInputCount += preStrippingCount;
-                        m_TotalVariantsOutputCount += (uint)inputData.Count;
-                        LogShaderVariants(shader, snippet, HDRenderPipelineGlobalSettings.instance.shaderVariantLogLevel, preStrippingCount, (uint)inputData.Count);
-                    }
```

Comment:
What was the point of this loop before? Seems like this would just have incorrectly multiplied the count values, and made duplicate logs. Good if it can be removed :)

Reply:
yeah, It didn't make sense at all :)